cmd/orb.go

```diff
@@ -84,7 +85,7 @@ func newOrbCommand() *cobra.Command {
 
 	orbCommand.AddCommand(orbPublishCommand)
 
-	orbCreateNamespace.PersistentFlags().StringVar(&organizationName, "org-name", "", "organization name")
+	orbCreateNamespace.PersistentFlags().StringVar(&organizationName, "org-name", "", "organization name (required)")
```

Comment:
You can use cobra to mark flags required:

```go
orbCreateNamespace.MarkPersistentFlagRequired("org-name")
```

Reply:
@marcomorain it doesn't look like `MarkPersistentFlagRequired` exists in Cobra. I used the same `panic` loop that you did in PR #35 ([see also](https://github.com/CircleCI-Public/circleci-cli/pull/35/files#diff-75d982f75dc436959ce23e09bf08d0fdR55)) and that works... but a little too aggressively. Now I get a panic result even when I _do_ include the flag. Any idea how to fix this? 

```
✗ go run main.go orb ns create sdf --org-name "circleci"
panic: no such flag -org-name

goroutine 1 [running]:
github.com/CircleCI-Public/circleci-cli/cmd.newOrbCommand(0xc420180000)
	/Users/hannah/go/src/github.com/CircleCI-Public/circleci-cli/cmd/orb.go:95 +0xcf4
github.com/CircleCI-Public/circleci-cli/cmd.MakeCommands(0x0)
	/Users/hannah/go/src/github.com/CircleCI-Public/circleci-cli/cmd/root.go:45 +0x2d4
github.com/CircleCI-Public/circleci-cli/cmd.Execute()
	/Users/hannah/go/src/github.com/CircleCI-Public/circleci-cli/cmd/root.go:21 +0x22
main.main()
	/Users/hannah/go/src/github.com/CircleCI-Public/circleci-cli/main.go:9 +0x20
exit status 2```